Add an option to run Content build with latest server

ronykoz opened the pull request:
## Status
- [ ] Ready
- [x] In code review
- [X] In Progress
- [ ] Hold
      Reason for hold:

## Description
Make Content build work with latest server Master

ronykoz commented:
DO NOT MERGE

yuvalbenshalom commented:
@ronykoz looks very good.
questions: 
1. where is the triggering script we should use to run the script? am i missing something?
2. why the mix with the integration files on the same branch?

got conflicts...

ronykoz commented:
@yuvalbenshalom  
1. My bad, added the script.
2. We have couple of problems with the validator script so it did not allow me to commit these files till i do these changes, added a possible fix but need to think this through

yuvalbenshalom commented on Utils/trigger_content_non_ami_build.sh:
```diff
@@ -0,0 +1,34 @@
+    post_data=$(cat <<EOF
+    {
+      "build_parameters": {
+        "AMI_RUN": "true"
```
should AMI_RUN be true in both cases?

ronykoz replied:
Actually it should be false, but on both cases

yuvalbenshalom commented:
resolve conflicts with integrations
